react-basic-practice/src/components/CustomerForm/index.tsx

```diff
@@ -22,11 +22,44 @@ const CustomerForm: React.FC<CustomerFormProps> = ({
   onSubmit,
   isLoading,
 }) => {
+  const [errors, setErrors] = useState<{ [key: string]: string }>({});
+
+  const validate = (name: string, value: string) => {
+    switch (name) {
+      case "name":
+        if (!value.trim()) return "Name is required";
```

Comment:
create error message constant. Update for all

Reply:
i done